lib/capabilities.go

```diff
@@ -78,3 +91,31 @@ func CapAwayNotify(caps *CapManager) {
 		},
 	)
 }
+
+func CapServerTime(caps *CapManager) {
+	name := "server-time"
+	caps.Supported[name] = ""
+
+	caps.FnsInitListener[name] = func(listener *Listener) {
+		listener.TagsEnabled = true
+	}
+
+	caps.FnsMessageToClient = append(
+		caps.FnsMessageToClient,
+		func(listener *Listener, message *ircmsg.IrcMessage) bool {
+			if !listener.IsCapEnabled(name) {
+				return false
+			}
+
+			_, exists := message.Tags["time"]
+			if !exists {
+				message.Tags["time"] = ircmsg.TagValue{
```

Comment:
If we wanna simplify this we can make it:
`message.Tags["time"] = ircmsg.MakeTagValue(time.Now().UTC().Format(time.RFC3339))`